FEXCore/Source/Interface/Core/JIT/VectorOps.cpp

```diff
@@ -529,7 +534,8 @@ DEF_OP(VFRSqrtScalarInsert) {
 
   auto ScalarEmitRPRES = [this, SubRegSize](ARMEmitter::VRegister Dst, std::variant<ARMEmitter::VRegister, ARMEmitter::Register> SrcVar) {
     auto Src = *std::get_if<ARMEmitter::VRegister>(&SrcVar);
-    frsqrte(SubRegSize.Scalar, Dst.D(), Src.D());
+    frsqrte(SubRegSize.Scalar, VTMP1.D(), Src.D());
+    ins(SubRegSize.Vector, Dst, 0, VTMP1, 0);
```

Comment:
What's with these added inserts? AFP.AH should change scalar operations to insert already? Including the additional two after these.

Reply:
Thanks for catching that. That was a mistake I made earlier on and I forgot to revert it. Apologies. Fixed now.